### PR TITLE
Release 7.4.0 & Bulk API Helm changes

### PR DESCRIPTION
**Pull request by mdebarros**
- Added Bulk Handler helm charts
- Upgraded the following Helm charts:
    - ml-api-adapter: v7.3.0
    - central-ledger: v7.3.1
    - central-event-processor: v7.1.1
    - simulator: v7.2.1
    - email-notifier: v7.3.0
    - central-settlements: v7.1.0 - app: v6.1.1
    - quoting-service: v7.4.0
    - mojaloop: v7.4.0
    - account-lookup-service: v7.4.0
- Added comprehensive health-checks to the following components:
    - ml-api-adapter
    - central-ledger
    - central-event-processor
    - email-notifier
- General maintenance
    - Added init container to ml-api-adapter service for kafka
    - Added new configs to support health-endpoint improvements
    - Aligned service ports for email-notifier
    - pullPolicy for Account-lookup is now `Always`
- Bug fixes:
    - Fixes for https://github.com/mojaloop/project/issues/918:
- **Important changes**
    - Made email-notifier disabled by default. If you require it, please set `emailnotifier.enabled=true`

### CODE REVIEW

File: mojaloop/values.yaml [hunk 1]
```diff
@@ -2642,7 +2657,7 @@ ml-api-adapter:
     service:
       type: ClusterIP
       externalPort: 80
-      internalPort: 8088
+      internalPort: 300
```
Comment (lewisdaly) [inline mojaloop/values.yaml]: Should this be `3000` as above?

Comment (mdebarros) [reply mojaloop/values.yaml]: Fixed in last push

File: account-lookup-service/values.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ account-lookup-service:
       image:
         repository: mojaloop/account-lookup-service
         tag: v6.2.4-snapshot
```
Comment (elnyry) [inline account-lookup-service/values.yaml]: Can we use the latest version here - v7.3.1-snapshot ?

File: account-lookup-service/values.yaml [hunk 3]
```diff
@@ -22,7 +22,7 @@ account-lookup-service:
...
```
Comment (elnyry) [inline account-lookup-service/values.yaml]: Can we use the latest version here - v7.3.1-snapshot ?

File: centralledger/chart-handler-transfer-position/values.yaml [hunk 4]
```diff
@@ -24,7 +24,7 @@ containers:
   api:
     image:
       repository: mojaloop/central-ledger
-      tag: v7.1.2
+      tag: vv7.3.1
```
Comment (elnyry) [inline centralledger/chart-handler-transfer-position/values.yaml]: typo

**Comment by elnyry** [comment]
Finally

image

**Pull request merged**